site/content/en/docs/handbook/troubleshooting.md

```diff
@@ -78,6 +78,37 @@ To view more detailed information about a pod, use:
 kubectl describe pod <name> -n <namespace>
 ```
 
+## Pods in error when using a container-based driver (docker, podman)
```

Comment:
this title could be more specific, and maybe a better place to put it would be FAQ ?
like "how to increase open file limits in minikube"
https://minikube.sigs.k8s.io/docs/faq/

Reply:
Agree that the title is too generic: there are plenty of reasons a Pod could err, we need to specify which one is addressed.
Also FAQ seem to have similar topics too. 
I will update the PR.